tests/transcriber/test_google_speech.py

```
@@ -0,0 +1,322 @@
+        # Save transcript
+        output_file = str(tmp_path / "transcript.txt")
+        transcript = "これはテストの文字起こしです。"
+
+        result = client.save_transcript(transcript, output_file)
+
+        # Verify file was created
+        assert os.path.exists(output_file)
+        assert result == output_file
+
+        # Verify content
+        with open(output_file, 'r', encoding='utf-8') as f:
```

Ruff (0.14.5)

Unnecessary mode argument

Remove mode argument

(UP015)